UpdateDB/DataLoading/ItemDataLoader.cs

```diff
@@ -1,255 +1,389 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Net.Http;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
-using HtmlAgilityPack;
 using log4net;
+using Newtonsoft.Json.Linq;
 using POESKillTree.Model.Items;
 using POESKillTree.Model.Items.Affixes;
 using POESKillTree.Model.Items.Enums;
-using POESKillTree.Utils;
 using POESKillTree.Utils.Extensions;
+using POESKillTree.Utils.WikiApi;
+
+using static POESKillTree.Utils.WikiApi.WikiApiUtils;
+using static POESKillTree.Utils.WikiApi.ItemRdfPredicates;
 
 namespace UpdateDB.DataLoading
 {
     /// <summary>
-    /// Extracts item bases from the wiki as a <see cref="XmlItemList"/>.
+    /// Retrieves item bases from the unofficial Wiki through its API.
```

Comment:
Technically it's linked to from pathofexile.com (see the "Game" dropdown in the menu bar), and it consequently calls itself the "Official Path of Exile Wiki".
This also appears in a few other places.